### ISSUE TITLE
May I have the opportunity to study your paper on STELLAR? I was unable to find your paper on Github.

### ISSUE THREAD

**Issue by Qzc11** [Vince-C156/PLANNING-AUTONOMOUS-SPACECRAFT-RENDEZVOUS-AND-DOCKING-TRAJECTORIES-VIA-REINFORCEMENT-LEARNING#6]

**Comment by Vince-C156** [comment]
Sure, I can email the paper.

**Comment by Qzc11** [comment]
I apologize, my previous email address has been forcefully suspended. Can you please send the paper to this email instead? [1715086956wang@gmail.com](mailto:1715086956wang@gmail.com). Thank you!

**Comment by Qzc11** [comment]
I'm very sorry, but I haven't received your email yet. My previous email account cannot be accessed due to security issues. Could you please help me by sending another copy to [1715086956wang@gmail.com](mailto:1715086956wang@gmail.com)? I would be extremely grateful!

**Comment by Vince-C156** [comment]
Yes, i'm not sure if I can just send it out. I'm asking to see if that's okay right now. Let me get back to you in 1-2 days.

**Comment by Qzc11** [comment]
Thank you for your help

**Comment by Vince-C156** [comment]
https://drive.google.com/file/d/1CF9YOZrl8GyAdmGA5gCzFVYm1tL34iBi/view

**Comment by Qzc11** [comment]
Thank you for your assistance. I wish you greater achievements in academic research.

**Comment by Vince-C156** [comment]
Let me know if you have any questions and please cite my work if you are planning to use this for academic purposes. That would be much appreciated!

**Comment by Qzc11** [comment]
Thank you for your assistance, and I will definitely cite your work if I use it for academic purposes.

**Comment by Qzc11** [comment]
Is the vbar-agentv3-3.pkl file in envstats pre-trained?

**Comment by Qzc11** [comment]
How do I need to use your code to get the kind of effect your paper achieves? I need your help.

**Comment by Vince-C156** [comment]
would you like to hop onto a zoom we can discuss this with more ease

**Comment by Qzc11** [comment]
I'd also like to ask you directly, but I'm sorry I'm not sure if Zoom can be used in China, does this code exactly correspond to what you published?

**Comment by Vince-C156** [comment]
Yes it does, however i'm not sure what your environment is.

**Comment by Qzc11** [comment]
My knowledge level is a little poor, and I really admire you for making such good results. Attached are some problems I do not understand, which may be very simple. I hope you can guide me, and I will be very grateful.
[问题.pptx](https://github.com/Vince-C156/STELLAR/files/14048843/default.pptx)

**Comment by Vince-C156** [comment]
I will reply the soonest possible. I currently work GNC for a space company so time is sparse. I'll try my best. Send me as many pictures of plots as you can!

**Comment by Qzc11** [comment]
Thank you so much for helping me in your busy schedule,I found that the environment script of the benchmark in STELLAR-ALPHA is not the same as the training environment script, and the reward script is also different, which environment should I use for cross-training?

**Comment by Vince-C156** [comment]
Can you send some images? I can reply by Thursday.

**Comment by Qzc11** [comment]
I'm sorry to see your reply until now for something recently. My English is a little poor, and I may not be able to describe some problems clearly, but I will try my best to express my problems. If you have time, please help me answer them, I will appreciate it very much.
[Uploading 问题.docx…]()

**Comment by Qzc11** [comment]
[问题.docx](https://github.com/Vince-C156/STELLAR/files/14127161/default.docx)

**Comment by Vince-C156** [comment]
Hi I have seen the document and I can respond in detail soon with another document.

**Comment by Vince-C156** [comment]
what will you be using my work for? Just out of curiosity.

**Comment by Qzc11** [comment]
Sorry for replying so late. I used it for my graduation project. Could you please send me the solution file? Thank you so much!

**Comment by Qzc11** [comment]
I would appreciate it if you have time to help me answer the question

**Comment by Qzc11** [comment]
Can you help me understand the pre-training process?

**Comment by Vince-C156** [comment]
You need to train it close range first and find success then take the same weights and try again from longer ranges slowly.

Sorry have been really busy doing some work for nasa. I don't think i'll be able to have time to fully help but if I do i will try my best.

**Comment by Qzc11** [comment]
I tried the pre-training task of x ∈ [−100, 100] m, y ∈ [400, 600] m, and z ∈ [−5, 5], adjusting the hyperparameters, but I didn't train the convergence curve, do I need to adjust the parameters of the reward or environment during the pre-training process?

**Comment by Qzc11** [comment]
![image](https://github.com/Vince-C156/STELLAR/assets/145653569/c7848b7f-7c04-4780-aeff-a09dd8be516f)

**Comment by Qzc11** [comment]
![image](https://github.com/Vince-C156/STELLAR/assets/145653569/be87fa5b-9915-4534-b8c9-dde8923fb2db)

**Comment by Qzc11** [comment]
This is the pre-trained curve, and I find that I always fail to win

**Comment by Qzc11** [comment]
![image](https://github.com/Vince-C156/STELLAR/assets/145653569/b24b9760-22b2-45df-b870-f3fa0cbcd453)
This is the hyperparameter, you see which setting is wrong, thanks

**Comment by Vince-C156** [comment]
can you provide some sample trajectories from runs that are associated with these parameters? The training cant be done with a single instance, you should run parallel instances to better generalize the gradiant updates.

**Comment by Qzc11** [comment]
[Uploading chaser5115.txt…]()

**Comment by Qzc11** [comment]
I want to confirm whether there is any problem with the setting of my reward environment parameters. Is the reward weight diag{.35.15.35 0.015 0.35 0.015}diag{10.5 10.5 10.5}? Is the pre-training task x ∈ [−100, 100] m, y ∈ [400, 600] m, and z ∈ [−5, 5], and what is the maximum time limit for the pre-training environment?
